### DISCUSSION TITLE
Welcome to Pulth Discussions!

### DISCUSSION THREAD

**Discussion by bekogeko** [Pulth-Team/Pulth#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dhincal** [comment]
It's my first time of using Github discussion and I agree with the idea of discussions helps our community to stick together and keep in touch!
